src/databricks/labs/blueprint/wheels.py

```diff
@@ -240,20 +279,20 @@ def _now_iso():
     def __enter__(self) -> "WheelsV2":
         """Builds the wheel and returns the instance. Use it as a context manager."""
         self._tmp_dir = tempfile.TemporaryDirectory()
-        self._local_wheel = self._build_wheel(self._tmp_dir.name, verbose=self._verbose)
+        self._local_wheel = self._build_wheel(self._tmp_dir.name, verbose=self._verbose, no_deps=self._no_deps)
```

Comment:
```suggestion
        self._local_wheel = next(self._build_wheel(self._tmp_dir.name, verbose=self._verbose, no_deps=True))
```
Local wheel type must remain the same. Turning it to a list makes no sense. No deps is the argument to function call here, remove it from constructor